mule-shaders-module/src/main/java/org/mule/module/shaders/api/ReflectiveAccessShader.java

```diff
@@ -0,0 +1,632 @@
+/*
+ * Copyright 2024 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.shaders.api;
```

Comment:
please review IDE warnings using the SONAR plugin.

Reply:
Since this file was copied from `org.apache.maven.plugins.shade.DefaultShader` and we have since confirmed with legal that we should open source the project under Apache v2.0, it makes more sense to keep this file as close to the original as possible, in order to make it easier to maintain for future versions of the plugin.